mongo/integration/unified/logger_verification.go

```diff
@@ -78,8 +74,20 @@ func newLogMessage(level int, msg string, args ...interface{}) (*logMessage, err
 // clientLogMessages is a struct representing the expected "LogMessages" for a
 // client.
 type clientLogMessages struct {
-	Client      string        `bson:"client"`
-	LogMessages []*logMessage `bson:"messages"`
+	Client         string        `bson:"client"`
+	IgnoreMessages []*logMessage `bson:"ignoreMessages"`
+	LogMessages    []*logMessage `bson:"messages"`
+}
+
+// ignore checks to see if the message is in the "IgnoreMessages" slice.
+func (clm clientLogMessages) ignore(ctx context.Context, msg *logMessage) bool {
+	for _, ignoreMessage := range clm.IgnoreMessages {
+		if err := verifyLogMatch(ctx, ignoreMessage, msg); err == nil {
+			return true
+		}
```

Comment:
This is a really unfortunate pattern, but I don't think there is a "set" solution that can be implemented here since the "logMessage" struct contains a slice of bytes. From the [comparison spec](https://go.dev/ref/spec#Comparison_operators):

> Slice, map, and function types are not comparable. 

Even though the time complexity is linear, the overall size of the IgnoreMessages should be very minimal.